src/panels/GettingStartedPanel/GettingStartedSteps.tsx

```diff
@@ -0,0 +1,242 @@
+    <FocusTask anchor={'Moon'} />
+    <CurrentFocus />
+    <Text>
+      Open the navigation menu and click on the "Focus" button to focus on the Moon. The
+      navigation menu can be found in the bottom bar:
+      <OriginPanelMenuButton onClick={() => {}} />
+    </Text>
+  </>,
+  <>
+    <Title order={2}>Let's go closer to the Moon!</Title>
+    <AltitudeTask anchor={'Moon'} altitude={3500} unit={'km'} compare={'lower'} />
+    <CurrentFocus />
+    <CurrentAltitude />
+  </>,
+  <Text>
+    Great! Now you know the basics of navigation. Let's start changing the time!
+  </Text>,
+  <>
+    <Text>Let's take a look at the Solar System from above.</Text>
+    <FocusTask anchor={'Sun'} />
+    <AltitudeTask anchor={'Sun'} altitude={1} unit={'Lighthours'} compare={'higher'} />
+    <NavigationTask anchor={'Sun'} lat={{ value: 80, min: 60, max: 100 }} />
+    <CurrentFocus />
+    <CurrentAltitude />
+    <CurrentLatLong />
+    <Text c={'dimmed'} fs={'italic'}>
+      The order of magnitue is km {'<'} AU {'<'} Lighthours
+    </Text>
+  </>,
+  <>
+    <Text>You can view the past as well as the future.</Text>
+    <ChangeYearTask />
+    <Text c={'dimmed'} fs={'italic'}>
+      You can reset the time by pressing "Now"
+    </Text>
+  </>,
+  <>
+    <Text>You can also change how fast time is playing.</Text>
+    <ChangeDeltaTimeTask />
+    <Text c={'dimmed'} fs={'italic'}>
+      You can reset the simulation speed by pressing "Real Time"
+    </Text>
+  </>,
+  <>
+    <Text>You can pause the time.</Text>
+    <PauseTimeTask />
+    <Text c={'dimmed'} fs={'italic'}>
+      Note: simulation speed is not set to 0 when you pause.
+    </Text>
+  </>,
+  <>
+    <Text>
+      Fantastic! Now you can move around in both space and time. Let's look at some
+      content!
+    </Text>
+  </>,
+  <>
+    <Text>All the content in OpenSpace can be found in the Scene menu.</Text>
+    <Group>
+      <SceneIcon size={IconSize.lg} /> Scene
+    </Group>
+    <Text>
+      You can search for objects in the top search bar and focus on objects by clicking
+      the focus icon <FocusIcon />.
+    </Text>
+  </>,
+  <>
+    You can turn on and off content by checking and unchecking their corresponding
+    checkboxes.
+    <SetBoolPropertyTask
+      uri={'Scene.EarthTrail.Renderable.Enabled'}
+      finalValue={false}
+      label={'Turn off the trail of Earth'}
+    />
+  </>,
+  <>
+    All objects have properties, which make it possible to alter the behavior or
+    appearance of an object.
+    <MarsTrailColorTask />
+  </>,
+  <>
+    <Text>Let's take a look at the surface of Mars.</Text>
+    <FocusTask anchor={'Mars'} />
+    <AltitudeTask anchor={'Mars'} altitude={10000} unit={'km'} compare={'lower'} />
+    <AltitudeMouse />
+  </>,
+  <>
+    <Text>
+      Planets can have multiple maps which show different data. Planets also have height
+      maps, which show elevation. Different maps can have different resolutions.
```

Comment:
Also more flavor text to explain a little bit more about maps, or if we link to our documentation page?